src/common/options/mds.yaml.in

```diff
@@ -1419,5 +1419,13 @@ options:
   type: bool
   level: dev
   default: false
+- name: mds_bal_rank_mask
+  type: str
+  level: advanced
+  desc: hexadecimal bit mask of the active MDS ranks to rebalance on.
+  long_desc: An MDS rank mask determines the group of active ranks on which it is suitable to rebalance.
```

Comment:
s/bit mask/bitmask/

Reply:
This method has been deprecated with the introduction of ceph fs set/get. Can we move them to somewhere?